src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -285,6 +297,30 @@ protected void interrupted() {
         };
     }
 
+    public Command UpSpecified(double inches) {
+        return new SubsystemCommand(this) {
+            double upSpecifiedIncrement = liftEncoder.get();
+            @Override
+            protected void initialize() {
+                disengageBrake();
+                upSpecifiedIncrement = liftEncoder.get();
+            }
+            @Override
+            protected void execute() {
+                if (upSpecifiedIncrement + 1 <=1) {
+                    liftDrive.set(.5);
+                } else liftDrive.set(0);
+            }
+            
+            @Override
+            protected boolean isFinished() {
+                if (upSpecifiedIncrement + 1 >=1) {
```

Comment:
This needs to look at the current encoder distance with liftEncoder.get() and compare it to the initial height + inches

Reply:
done